planning/behavior_path_planner/src/planner_manager.cpp

```diff
@@ -639,6 +639,23 @@ void PlannerManager::print() const
     return;
   }
 
+  const auto get_status = [](const auto & m) {
+    const auto status = m->getCurrentStatus();
+    if (status == ModuleStatus::RUNNING) {
+      return "RUNNING";
+    }
+
+    if (status == ModuleStatus::FAILURE) {
+      return "FAILURE";
+    }
+
+    if (status == ModuleStatus::SUCCESS) {
+      return "SUCCESS";
+    }
+
+    return "NONE";
+  };
+
```

Comment:
FYI
magic_enum::enum_name can covert enum to string

Reply:
Use it.

Reply:
:+1: